pom.xml

```diff
@@ -448,5 +448,12 @@
 			<version>2.0.11-beta</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>

```

Comment:
This dependency doesn't seem to be used in your test. Nothing calls com.fiftyonred.SOMETHING
Is it needed?